tests/trial.py

```diff
@@ -1,10 +1,10 @@
 from pyannote.database import get_protocol
 
-protocol = get_protocol('MyDatabase.SpeakerVerification.MySpeakerVerification')
+protocol = get_protocol("MyDatabase.SpeakerVerification.MySpeakerVerification")
```

Comment:
Looks like change in `tests/trial.py` are unrelated to this PR main topic.
Can you revert those changes?